submissions/Shramkoweb/html-css-popup/source/sass/blocks/visually-hidden.scss

```diff
@@ -0,0 +1,13 @@
+.visually-hidden {
+  position: absolute;
+
+  width: 1px;
+  height: 1px;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+
+  border: 0;
+
+  clip: rect(1px, 1px, 1px, 1px);
```

Comment:
CSS-Tricks recommend to use `clip: rect(0 0 0 0); `.

Reply:
Thx.  Well, that's... that's a matter of opinion.